styles/lesson.less

```diff
@@ -289,6 +295,35 @@ input[type="text"] {
   margin-top: 10px;
 }
 
+input[type="checkbox"] {
+  display: none;
+}
+
+input[type="checkbox"] + label {
```

Comment:
`<label>` which is sibling to `<input type="checkbox">`.

https://stackoverflow.com/questions/1139763/what-does-the-plus-sign-css-selector-mean